www/docs/install.md

```diff
@@ -76,13 +76,26 @@ All artifacts are checksummed, and the checksum is signed with [cosign][].
 	```sh
 	wget https://github.com/goreleaser/nfpm/releases/download/__VERSION__/checksums.txt
 	wget https://github.com/goreleaser/nfpm/releases/download/__VERSION__/checksums.txt.sig
+	wget https://github.com/goreleaser/nfpm/releases/download/__VERSION__/checksums.txt.pem
 	```
 
 1. Verify the signature:
 	```sh
-	COSIGN_EXPERIMENTAL=1 cosign verify-blob \
+	cat checksums.txt.pem | base64 -d | openssl x509 -text -noout
```

Comment:
I need to check the new cosign release instructions, but, assuming this is right, we need to use the `__VERSION__` var instead of an specific version.